monkey/infection_monkey/post_breach/pba.py

```diff
@@ -45,7 +45,10 @@ def run(self):
         """
         exec_funct = self._execute_default
         result = exec_funct()
-        hostname = socket.gethostname()
+        try:
+            hostname = socket.gethostname()
+        except socket.error:
+            hostname = ""
```

Comment:
We will then get `0.0.0.0`. This will confuse us later on. Can we find a better poison value? Or do we want later on code to account for this?

Reply:
Hostname and IP are only used in telemetry brief. For now I changed them to "Unknown"